zajecia/src/time/Main.java

```diff
@@ -0,0 +1,37 @@
+package time;
+
+import java.time.*;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
prosba o wrzucenie tego do osobnych metod ktore beda wywolowane w mainie, to podejscie jest takim wstepem do pisania testow jednostkowych:
`
main() {
testZad1();
testZad2();
testZad3();
...
}

void testZad1 (){
...
}
`